source/overloading.tex

```diff
@@ -3440,331 +3442,295 @@
 
 \begin{codeblock}
-@\textit{VQ T}@& operator--(@\textit{VQ T}@&);
-@\textit{T}@ operator--(@\textit{VQ T}@&, int);
+@\cvqual{vq} \placeholder{T}@& operator--(@\cvqual{vq} \placeholder{T}@&);
+@\placeholder{T}@ operator--(@\cvqual{vq} \placeholder{T}@&, int);
 \end{codeblock}
 
 \pnum
 For every pair
-(\textit{T},
-\textit{VQ}),
+(\tcode{\placeholder{T}},
+\cvqual{vq}),
 where
-\textit{T}
-is a cv-qualified or cv-unqualified object type, and
-\textit{VQ}
-is either
-\tcode{volatile}
-or empty,
+\tcode{\placeholder{T}}
+is a cv-qualified or cv-unqualified object type,
 there exist candidate operator functions of the form
 
 \begin{codeblock}
-@\textit{T}@*@\textit{VQ}@& operator++(@\textit{T}@*@\textit{VQ}@&);
-@\textit{T}@*@\textit{VQ}@& operator--(@\textit{T}@*@\textit{VQ}@&);
-@\textit{T}@*    operator++(@\textit{T}@*@\textit{VQ}@&, int);
-@\textit{T}@*    operator--(@\textit{T}@*@\textit{VQ}@&, int);
+@\placeholder{T}@*@\cvqual{vq}@& operator++(@\placeholder{T}@*@\cvqual{vq}@&);
+@\placeholder{T}@*@\cvqual{vq}@& operator--(@\placeholder{T}@*@\cvqual{vq}@&);
+@\placeholder{T}@*    operator++(@\placeholder{T}@*@\cvqual{vq}@&, int);
+@\placeholder{T}@*    operator--(@\placeholder{T}@*@\cvqual{vq}@&, int);
 \end{codeblock}
 
 \pnum
 For every cv-qualified or cv-unqualified object type
-\textit{T},
+\tcode{\placeholder{T}},
 there exist candidate operator functions of the form
 
 \begin{codeblock}
-@\textit{T}@&    operator*(@\textit{T}@*);
+@\placeholder{T}@&    operator*(@\placeholder{T}@*);
 \end{codeblock}
 
 \pnum
 For every function type
-\textit{T} that does not have cv-qualifiers or a \grammarterm{ref-qualifier},
+\tcode{\placeholder{T}} that does not have cv-qualifiers or a \grammarterm{ref-qualifier},
 there exist candidate operator functions of the form
 
 \begin{codeblock}
-@\textit{T}@&    operator*(@\textit{T}@*);
+@\placeholder{T}@&    operator*(@\placeholder{T}@*);
 \end{codeblock}
 
 \pnum
-For every type \textit{T} there exist candidate operator functions of the form
+For every type \tcode{\placeholder{T}} there exist candidate operator functions of the form
 
 \begin{codeblock}
-@\textit{T}@*    operator+(@\textit{T}@*);
+@\placeholder{T}@*    operator+(@\placeholder{T}@*);
 \end{codeblock}
 
 \pnum
 For every promoted arithmetic type
-\textit{T},
+\tcode{\placeholder{T}},
 there exist candidate operator functions of the form
 
 \begin{codeblock}
-@\textit{T}@ operator+(@\textit{T}@);
-@\textit{T}@ operator-(@\textit{T}@);
+@\placeholder{T}@ operator+(@\placeholder{T}@);
+@\placeholder{T}@ operator-(@\placeholder{T}@);
 \end{codeblock}
 
 \pnum
 For every promoted integral type
-\textit{T},
+\tcode{\placeholder{T}},
 there exist candidate operator functions of the form
 
 \begin{codeblock}
-@\textit{T}@ operator~(@\textit{T}@);
+@\placeholder{T}@ operator~(@\placeholder{T}@);
 \end{codeblock}
 
 \pnum
 For every quintuple
-(\textit{C1},
-\textit{C2},
-\textit{T},
-\textit{CV1},
-\textit{CV2}),
+(\tcode{\placeholder{C1}},
```

Comment:
*Now* we could start thinking about subscripting, e.g. `\tcode{\placeholder{C}}$_1$`. But we can do that in a separate iteration.